llmfoundry/models/__init__.py

```diff
@@ -24,6 +25,7 @@
 models.register('fmapi_causal_lm', func=FMAPICasualLMEvalWrapper)
 models.register('openai_chat', func=OpenAIChatAPIEvalWrapper)
 models.register('fmapi_chat', func=FMAPIChatAPIEvalWrapper)
+tp_strategy.register('ffn', func=ffn_tp_strategy)
```

Comment:
I'd prefer not to stick the tp_strategies in utils. lets make a `llmfoundry/tp` folder, and `tp_strategies.py` in there?

Reply:
done